langkit/stylechecks/__init__.py

```diff
@@ -452,25 +452,36 @@ def check(self, report, filename, content, parse):
         self.custom_check(report, filename, content, parse)
         if os.path.exists(filename):
             self.pep8_check(report, filename)
-            self.pyflakes_check(report, filename, content)
+            # self.pyflakes_check(report, filename, content)
```

Comment:
Is this permanent? Can you explain?

Reply:
It's a mistake ... Or the reason code reviews exists :) Thanks !